plugin/src/api/index.ts

```diff
@@ -52,9 +59,57 @@ export class TodoistApiClient {
     return camelize(JSON.parse(response.body)) as Label[];
   }
 
+  /**
+   * Gets a list of completed tasks from Todoist using the sync API.
+   */
+  public async getCompletedTasks(params?: GetCompletedTasksParams): Promise<CompletedTask[]> {
+    const queryParams = new URLSearchParams();
+
+    if (params) {
+      if (params.project_id) queryParams.set("project_id", params.project_id);
+      if (params.limit) queryParams.set("limit", params.limit.toString());
+      if (params.until) queryParams.set("until", params.until.toISOString());
+      if (params.since) queryParams.set("since", params.since.toISOString());
+    }
+
+    queryParams.set("annotate_items", "true");
+
+    const url = `${this.syncBaseUrl}/completed/get_all${
+      queryParams.toString() ? `?${queryParams.toString()}` : ""
+    }`;
+
+    debug({
+      msg: "Sending Todoist API request",
+      context: Object.fromEntries(queryParams.entries()),
+    });
```

Comment:
Can we debug log the `RequestParams` that gets passed into `fetcher.fetch` like what happens in `do`?